docs/changelog/1_0_b1.rst

```diff
@@ -0,0 +1,332 @@
+==========
+1.0 Beta 1
+==========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 beta 1 "Sirius".
+
+
+Migrations
+==========
+
+One of the major improvements in Beta release is the migrations
+support (`RFC 1000 <migrations_>`_). It is now possible to manage
```

Comment:
This reads strange, since migrations have been there for a while. I'd rephrase to something along the "This Beta release features the fully implemented RFC 1000 for the first time"

Reply:
Fair point, I'll update the wording.